libvmi/os/windows/core.c

```diff
@@ -414,6 +414,16 @@ get_kpgd_method0(
     return VMI_FAILURE;
 }
 
+status_t windows_get_kernel_struct_offset(vmi_instance_t vmi, const char* symbol, const char* subsymbol, addr_t *addr){
+    status_t ret = VMI_SUCCESS;
+    windows_instance_t windows = vmi->os_data;
+    if (VMI_FAILURE == rekall_profile_symbol_to_rva(windows->rekall_profile,symbol,subsymbol,addr)){
```

Comment:
You can just return the call to rekall_profile_symbol_to_rva directly, no need, for the local variable ret.